src/lwc/geTemplateBuilder/geTemplateBuilder.js

```diff
@@ -258,6 +264,7 @@ export default class geTemplateBuilder extends NavigationMixin(LightningElement)
             this.buildBatchTableColumnOptions(this.formSections);
             this.setInitialActiveFormSection();
 
+            this.isElevateCustomer = await checkForElevateCustomer();
```

Comment:
I think since `geTemplateBuilderService` needs to know if a customer is an elevate customer and we use it in `geTemplateBuilder`, we can expose that information instead of making another call here to the backend for this information. Long-term I would want to see this information cached at the top level and accessible to all the child components that need it.

Reply:
Nice catch, updated! Thanks!